src/test/java/io/confluent/idesidecar/restapi/models/graph/RealLocalFetcherTest.java

```diff
@@ -191,4 +195,51 @@ void assertExtractLocalhostAddresses(String input, String...expectedAddresses) {
     assertEquals(Arrays.asList(expectedAddresses), addresses);
   }
 
+  @Test
+  void schemaRegistryFetch_shouldParseConfigResponseFromValidJson() {
+    var json = loadResource(
+        "confluent-local-resources-mock-responses/get-schema-registry-response.json"
+    );
+    SchemaRegistryConfigResponse config = localFetcher.parseSchemaRegistryConfig(URL, json);
+    assertEquals("BACKWARD", config.compatibilityLevel());
+
+  }
+
+  @Test
+  void shouldReturnValidSchemaRegistryUri() throws CreateConnectionException {
+    var localConfig = new ConnectionSpec.LocalConfig("http://localhost:8085");
+    var connectionSpec = new ConnectionSpec("1", "Local Connection", ConnectionSpec.ConnectionType.LOCAL, null, localConfig);
+    manager.createConnectionState(connectionSpec);
+
+    String uri = localFetcher.resolveSchemaRegistryUri("1");
+    assertEquals("http://localhost:8085", uri);
+  }
+
+  @Test
+  void shouldReturnDefaultSchemaRegistryUriWhenLocalConfigIsNull() throws CreateConnectionException {
+    var connectionSpec = new ConnectionSpec("2", "Local Connection", ConnectionSpec.ConnectionType.LOCAL, null, null);
+    manager.createConnectionState(connectionSpec);
+
+    String uri = localFetcher.resolveSchemaRegistryUri("2");
+    assertEquals("http://localhost:8081", uri);
+  }
+
+  @Test
+  void shouldReturnNullWhenSchemaRegistryUriIsBlank() throws CreateConnectionException {
+    var localConfig = new ConnectionSpec.LocalConfig("");
+    var connectionSpec = new ConnectionSpec("3", "Local Connection", ConnectionSpec.ConnectionType.LOCAL, null, localConfig);
+    manager.createConnectionState(connectionSpec);
+
+    String uri = localFetcher.resolveSchemaRegistryUri("3");
+    assertEquals("", uri); // No Schema Registry
```

Comment:
I understood this `resolveSchemaRegistryUri()` method to get the URI schema registry that we would use, or null if there is none. After all, that's what [this method is doing](https://github.com/confluentinc/ide-sidecar/pull/62/files#diff-7822e65aad428c179cdf0d984e6523c78578616f77d2e6c162cc937caf258eabR164).

I think this test is wrong because my suggestion for the `resolveSchemaRegistryUri()` method was wrong (see earlier comment in this review).

```suggestion
    String uri = localFetcher.resolveSchemaRegistryUri("3");
    assertNul(uri); // No Schema Registry
```

Reply:
Done.